hbw/app/javascript/packs/hbw/connection.js

```diff
@@ -216,12 +216,20 @@ modulejs.define('HBWConnection', ['jQuery'], (jQuery) => {
         this.options.host = `${document.location.protocol}//${document.location.host}`;
       }
 
+      this.payload = options.payload;
```

Comment:
Hi! ✋ 
You have no reason to do it, payload is already in `this.options.payload` set at line 210.